clap_builder/src/builder/styled_str.rs

```diff
@@ -208,3 +210,9 @@ impl std::fmt::Display for StyledStr {
         Ok(())
     }
 }
+
+impl From<Cow<'static, str>> for StyledStr {
+    fn from(value: Cow<'static, str>) -> Self {
+        StyledStr(value.to_string())
```

Comment:
Please use `.into_owned()`

`to_string` can work on any `display` and can make code more sloppy on refactor.

(sorry, I overlooked this last time and apparently a clippy lint didn't catch this case)

Reply:
No, `Cow<'static, str>::to_string` will cause unnecessary clone on owned case, `into_owned` is indeed what we want.